console.go

```diff
@@ -96,6 +97,7 @@ func NewConsoleWriter(options ...func(w *ConsoleWriter)) ConsoleWriter {
 
 // Write transforms the JSON input with formatters and appends to w.Out.
 func (w ConsoleWriter) Write(p []byte) (n int, err error) {
+	mu.Lock()
```

Comment:
This lock is ineffective as there is no way for the potential writer of those props to acquire it.